packages/sampleCommons/config/cbds/explorer.json

```diff
@@ -44,45 +44,76 @@
         "disability_adjusted_life_years"
       ]
     },
+    "dropdowns": {
+      "download-table": {
+        "title": "Download Table"
+      }
+    },
+    "buttons": [
+      {
+        "enabled": true,
+        "type": "data-json",
+        "title": "JSON",
+        "dropdownId": "download-table",
+        "leftIcon": "datafile",
+        "rightIcon": "download",
+        "fileName": "MIDRC_Measurements_table.json"
+      },
+      {
+        "enabled": true,
+        "type": "data-tsv",
+        "title": "TSV",
+        "dropdownId": "download-table",
+        "leftIcon": "datafile",
+        "rightIcon": "download",
+        "fileName": "MIDRC_Measurements_table.tsv"
+      },
+      {
+        "enabled": true,
+        "type": "data-csv",
+        "title": "CSV",
+        "dropdownId": "download-table",
+        "leftIcon": "datafile",
+        "rightIcon": "download",
+        "fileName": "MIDRC_Measurements_table.csv"
+      }
+    ],
+    "loginForDownload": true,
     "guppyConfig": {
-      "dataType": "patient",
+      "dataType": "measurement",
+      "nodeCountTitle": "Measurements",
       "fieldMapping": [
         {
-          "field": "quality_adjusted_life_years",
-          "name": "quality_adjusted_life_years"
+          "field": "project_id",
+          "name": "Project ID"
+        },
+        {
+          "field": "submitter_id",
+          "name": "Measurement ID"
         },
         {
-          "field": "disability_adjusted_life_years",
-          "name": "disability_adjusted_life_years"
+          "field": "case_ids",
+          "name": "Case ID"
         }
       ],
-      "nodeCountTitle": "Patients",
-      "manifestMapping": {
-        "resourceIndexType": "Patient",
-        "resourceIdField": "id",
-        "referenceIdFieldInResourceIndex": "id",
-        "referenceIdFieldInDataIndex": "id"
-      },
-      "accessibleFieldCheckList": [
-        "project_id"
-      ],
+      "accessibleFieldCheckList": ["project_id"],
       "accessibleValidationField": "project_id"
     }
   },
   {
     "tabTitle": "Observation",
     "charts": {
       "project_id": {
-        "chartType": "fullPie",
+        "chartType": "donut",
```

Comment:
might need to resolve some of these and keep our changescfor this file... not sure which is which here